src/TicketContext/TicketContext.js

```diff
@@ -0,0 +1,45 @@
+import React, { createContext, useState, useEffect } from 'react';
+
+export const TicketContext = createContext();
```

Comment:
Can use `null` as the default value https://react.dev/reference/react/createContext#parameters